frontend/packages/ceph-storage-plugin/OWNERS

```diff
@@ -5,11 +5,13 @@ reviewers:
   - cloudbehl
   - gnehapk
   - rawagner
+  - shyRozen
```

Comment:
Can you create `OWNERS` file inside the integration-tests folder and give access to reviewers and Approvers over there.